services/core/cards-consultation/src/test/java/org/opfab/cards/consultation/application/configuration/ThreadPoolTaskSchedulerConfiguration.java

```diff
@@ -0,0 +1,22 @@
+/* Copyright (c) 2021, RTE (http://www.rte-france.com)
+ * See AUTHORS.txt
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ * SPDX-License-Identifier: MPL-2.0
+ * This file is part of the OperatorFabric project.
+ */
+package org.opfab.cards.consultation.application.configuration;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.scheduling.concurrent.ThreadPoolTaskScheduler;
+
+@Configuration
+public class ThreadPoolTaskSchedulerConfiguration {
+    
+    @Bean
+    public ThreadPoolTaskScheduler taskScheduler(){
+        return new ThreadPoolTaskScheduler();
+    }
+}
```

Comment:
Instead of creating a duplicate configuration class for the tests I think you can just import the "real" configuration in `services/core/cards-consultation/src/test/java/org/opfab/cards/consultation/application/IntegrationTestApplication.java` using the @Import annotation.
This way there is no risk of the test configuration not being the same as the real one.

Reply:
Done